src/course-outline/CourseOutline.test.jsx

```diff
@@ -631,9 +631,9 @@ describe('<CourseOutline />', () => {
       .onGet(getXBlockApiUrl(section.id))
       .reply(200, section);
 
-    const [firstSection] = await findAllByTestId('section-card');
+    const [firstSubsection] = await findAllByTestId('section-card');
```

Comment:
Why rename this to subsection?

Reply:
Nice catch! It seems treesitter rename is not perfect and it changed words outside the test. Fixed it.